### PR TITLE
Navigation menu

### LINKED ISSUE #70: Implement header menu

### PR DESCRIPTION
**Pull request by Danik1702**
- add header navigation menu 
- delete local logos in all pages

### CODE REVIEW

**Review by ivan-bonk — APPROVED** [review]
Big work 👍 
good job

**Review by Pashkaua — APPROVED** [review]
Nice !

File: src/pages/pet/pet.component.tsx [hunk 1]
```diff
@@ -39,9 +37,6 @@ export const Pet: React.FC<PetProps> = props => {
   //TODO: Skeleton or placeholder. Add during loading the page @O.Khabrovska
   return (
     <Fragment>
-      <div className={style.logoContainer}>
-        <Logo />
```
Comment (okhabr) [inline src/pages/pet/pet.component.tsx:48]: Thanks for taking care of all the pages 👍

**Review by okhabr — APPROVED** [review]
🥇

**Pull request merged**